src/manager/manager.go

```diff
@@ -162,6 +162,14 @@ func (manager *Manager) Start(ctx context.Context) error {
 		manager.janitorSubscriber.Subscribe(manager.eventBus)
 		go manager.janitorServer.Init().Run()
 	}
+	// send proxy info to dns proxy listener
```

Comment:
put the following block under if statement, only EnableProxy set true then we add this DNS record